pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -178,7 +186,12 @@ stdenv.mkDerivation rec {
     ++ lib.optional smbdSupport "--smbd=${samba}/bin/smbd"
     ++ lib.optional uringSupport "--enable-linux-io-uring"
     ++ lib.optional canokeySupport "--enable-canokey"
-    ++ lib.optional capstoneSupport "--enable-capstone";
+    ++ lib.optional capstoneSupport "--enable-capstone"
+    ++ lib.optional (!pluginsSupport) "--disable-plugins"
+    ++ lib.optional (!enableBlobs) "--disable-install-blobs"
+
+    # FIXME: "multiple definition of `strtoll'" with libnbcompat
+    ++ lib.optional stdenv.hostPlatform.isStatic "--static --extra-ldflags=-Wl,--allow-multiple-definition";
```

Comment:
Would be nice if we could figure out why this is happening.  One definition is from libnbcompat — where's the other from?

Reply:
Seems to be a conflict between libnbcompat and musl:

```
/nix/store/kbhvb5y763zm2xskz94j176qv2dvbv58-x86_64-unknown-linux-musl-binutils-2.40/bin/x86_64-unknown-linux-musl-ld: /nix/store/g29g3hydcrrjkisrmk96gnx23mnnldrj-musl-static-x86_64-unknown-linux-musl-1.2.3/li
b/libc.a(strtol.lo): in function `strtoll.localalias':                                                  
(.text.strtoll+0x0): multiple definition of `strtoll'; /nix/store/dp31apbsw47a9bm37w45kh96vkngh2az-compat-netbsd-static-x86_64-unknown-linux-musl-9.2/lib/libnbcompat.a(strtoll.o):(.text+0x0): first defined he
re
```

Also it appears that there are many more build dependencies that we can remove. I'll play around and see how much further we can reduce the input closure.

Reply:
Thanks.  I wonder why we don't see this on other packages.  I'd like to take a look, but don't know when I'll have the capacity…